LanguageFeatures/Control-flow-collections/Null-aware-elements/warnings_A01_t02.dart

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion As we do for other null-aware expressions like `?.` and `...?`,
+/// compilers and IDEs should probably warn if the inner expressions in
+/// null-aware elements are not potentially nullable since in that case, the `?`
+/// has no meaningful effect.
+///
+/// @description Check that it is a warning if the inner expressions in
+/// null-aware elements are not potentially nullable. Test literals
+/// @author sgrekhov22@gmail.com
+
+main() {
+  <Object>{
+    ?42,
+//  ^
+// [analyzer] unspecified
+// [cfe] unspecified
+    ?3.14,
+//  ^
+// [analyzer] unspecified
+// [cfe] unspecified
+    ?true,
+//  ^
+// [analyzer] unspecified
+// [cfe] unspecified
+    ?"",
+//  ^
+// [analyzer] unspecified
+// [cfe] unspecified
+    ?'''  Raw string   '''
```

Comment:
Perhaps:
```suggestion
    ?'''  Triple quoted string   '''
```
because 'raw string' has a specific meaning. However, we might also want to have `? r'Actually a raw string'`.

Reply:
Fixed and added true raw string

Reply:
Thx!